src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/BasicHtmlWebResponseObject.Common.cs

```diff
@@ -96,7 +92,7 @@ public WebCmdletElementCollection Links
                 if (_links == null)
                 {
                     List<PSObject> parsedLinks = new();
-                    MatchCollection linkMatch = HtmlParser.LinkRegex.Matches(Content);
+                    MatchCollection linkMatch = HtmlParser.LinkRegex.Matches(Content!);
```

Comment:
Again, `!` usually says that something is wrong.
Please annotate `InitializeContent` method with [MemberNotNull](https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.codeanalysis.membernotnullattribute)

Reply:
done